config/initializers/cors.rb

```diff
@@ -5,12 +5,15 @@
 
 # Read more: https://github.com/cyu/rack-cors
 
-# Rails.application.config.middleware.insert_before 0, Rack::Cors do
-#   allow do
-#     origins 'example.com'
-#
-#     resource '*',
-#       headers: :any,
-#       methods: [:get, :post, :put, :patch, :delete, :options, :head]
-#   end
-# end
+Rails.application.config.middleware.insert_before 0, Rack::Cors do
+  allow do
+    origins 'http://localhost:3001'
```

Comment:
@AlexMMcConnell @larroyo1  Hi 👋  Sorry to jump in with a review on code that has already been merged, but somehow this was switched again.  Front end is running on port 3000 please!

Reply:
Screenshot from @ivonne-hernandez 
<img width="1433" alt="Screen Shot 2022-02-08 at 3 01 49 PM" src="https://user-images.githubusercontent.com/724355/153121691-c73b4d95-23f5-4d17-a35d-8710141124ab.png">